src/wp-includes/class-wp-block.php

```diff
@@ -226,7 +226,14 @@ public function render( $options = array() ) {
 			wp_enqueue_style( $this->block_type->style );
 		}
 
-		/** This filter is documented in src/wp-includes/blocks.php */
+		/**
+		 * Filters the content of a single block.
```

Comment:
Was the rest of this file update in another PR?

Reply:
Yes, it was already backported as part of https://core.trac.wordpress.org/ticket/49926:
- `WP_Block` class + tests
- `WP_Block_List` class + tests